From 2112c4a71e12d4b69410a01d6643769b2346fd03 Mon Sep 17 00:00:00 2001
From: Bai Ping <ping.bai@nxp.com>
Date: Tue, 5 Sep 2017 11:13:12 +0800
Subject: [PATCH 2507/5242] MLK-16367 driver: clk: imx: enable ddrc apb clock
 always on i.mx8mq

commit  a622138ce0126bb115f8062b53de6e8b9e6266a2 from
https://source.codeaurora.org/external/imx/linux-imx.git

Keep the DDRC APB init on, so we can access the DDRC register.

Signed-off-by: Bai Ping <ping.bai@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/clk/imx/clk-imx8mq.c |    1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/clk/imx/clk-imx8mq.c b/drivers/clk/imx/clk-imx8mq.c
index 2728aad..165d0b3 100644
--- a/drivers/clk/imx/clk-imx8mq.c
+++ b/drivers/clk/imx/clk-imx8mq.c
@@ -281,6 +281,7 @@
 	IMX8MQ_CLK_USB_BUS_CG, IMX8MQ_CLK_NAND_USDHC_BUS_CG,
 	IMX8MQ_CLK_MAIN_AXI_CG, IMX8MQ_CLK_A53_CG,
 	IMX8MQ_CLK_AUDIO_AHB_DIV, IMX8MQ_CLK_TMU_ROOT,
+	IMX8MQ_CLK_DRAM_APB_DIV,
 };
 
 static struct clk_onecell_data clk_data;
-- 
1.7.9.5

